XenoHolographic-it/IAR-SDK-Sample-Android
# Feature/IAR-4919

# Overview

Add NFC-read screen and implement read NFC functionality

# Resources

https://iversoft.atlassian.net/browse/IAR-4919

# Technical Considerations

-

# Known Issues

-

# Testing Instructions

* Click ‘Read NFC Tag’ button, Scan a NFC tag, to see if there is ‘Read NFC successfully” message and the correct marker id.
* Click ‘View the Scanned Marker”, to see if navigate to surface view

# Screenshots
![Screenshot_4919](https://user-images.githubusercontent.com/96147375/152437304-d3f8a7ca-87b6-4250-8209-43a82c980cab.jpg)
